src/cli/cli_dataset.cpp

```diff
@@ -151,6 +166,34 @@ template <> otError Dataset::Process<Cmd("init")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli dataset active
+ * @code
+ * dataset active
+ * Active Timestamp: 1
+ * Channel: 13
+ * Channel Mask: 0x07fff800
+ * Ext PAN ID: d63e8e3e495ebbc3
+ * Mesh Local Prefix: fd3d:b50b:f96d:722d::/64
+ * Network Key: dfd34f0f05cad978ec4e32b0413038ff
+ * Network Name: OpenThread-8f28
+ * PAN ID: 0x8f28
+ * PSKc: c23a76e98f1a6483639b1ac1271e2e27
+ * Security Policy: 0, onrcb
+ * Done
+ * @endcode
+ * @code
+ * dataset active -x
+ * 0e080000000000010000000300001035060004001fffe002084eb74ab03c56e6d00708fdc7fe165c83a67805108e2104f183e698da87e96efc1e45aa51030f4f70656e5468726561642d383631310102861104108d6273023d82c841eff0e68db86f35740c030000ff
```

Comment:
This is causing `pretty` check to fail. Is there any way we can include line breaks here that are removed as part of the conversion?

Reply:
I added the common `...` indicator to indicate a trimmed string - will that work?